lib/core/gonzales.js

```diff
@@ -1,16 +1,14 @@
 import gonzales from 'gonzales-pe';
-import gonzalesPrimitive from '../../packages/gonzales-primitives';
 
+/**
+ * A wrapper to prevent failure on parsing error.
+ */
 export default {
   parse: (...args) => {
     try {
       return gonzales.parse(...args);
     } catch (e) {
-      try {
-        return gonzalesPrimitive.parse(...args);
```

Comment:
Why is this no longer needed?

Reply:
It's mentioned in the PR description: 

Update gonzales-pe to v4.3.0 which fixed a [parsing bug](https://github.com/atfzl/eslint-plugin-css-modules/blob/7cddaceb1010427fd9c1ddaf0e346abbbcd6bd08/test/lib/rules/no-undef-class.js#L250-L261) and [more](https://github.com/tonyganch/gonzales-pe/compare/v4.1.0...v4.2.4).
    Remove gonzales-primitives that was previously added as [a bandaid](https://github.com/atfzl/eslint-plugin-css-modules/commit/737d4494561af8dc7ba57973383b72c147492139) for that parsing bug.

Reply:
`gonzalesPrimitive` is an old `gonzales-pe` version that was [added by the previous maintainer](https://github.com/atfzl/eslint-plugin-css-modules/commit/737d4494561af8dc7ba57973383b72c147492139), possibly to address some bugs introduced in the version of `gonzales-pe` used at that time (`v4.0.3`).

With the latest version `gonzales-pe v4.3.0`, [a lot of bugs were fixed](https://github.com/atfzl/eslint-plugin-css-modules/pull/83#:~:text=parsing%20bug%20and-,more,-.) and the [test case added](https://github.com/atfzl/eslint-plugin-css-modules/blob/737d4494561af8dc7ba57973383b72c147492139/test/lib/rules/no-undef-class.js#L131-L142) was no longer failing. Therefore, I'm removing this fallback after upgrading to `v4.3.0`.